deepchecks/nlp/utils/text_properties.py

```diff
@@ -55,16 +55,32 @@ def get_transformer_model(
     property_name: str,
     model_name: str,
     device: Optional[str] = None,
-    quantize_model: bool = False
+    quantize_model: bool = False,
+    models_storage: Union[pathlib.Path, str, None] = None
 ):
     """Get the transformer model and decide if to use optimum.onnxruntime.
 
     optimum.onnxruntime is used to optimize running times on CPU.
     """
+    if models_storage is None:
+        models_storage = MODELS_STORAGE
+    else:
+        if isinstance(models_storage, str):
+            models_storage = pathlib.Path(models_storage)
+        if not isinstance(models_storage, pathlib.Path):
+            raise ValueError(
+                f'Unexpected type of the "models_storage" parameter - {type(models_storage)}'
+            )
+        if not models_storage.is_dir():
+            raise ValueError('"model_storage" expected to be a directory')
+
     if device not in (None, 'cpu'):
         transformers = _import_optional_property_dependency('transformers', property_name=property_name)
         # TODO: quantize if 'quantize_model' is True
-        return transformers.AutoModelForSequenceClassification.from_pretrained(model_name)
+        return transformers.AutoModelForSequenceClassification.from_pretrained(
+            model_name,
+            cache_dir=models_storage
```

Comment:
maybe also change the argument name in our function to cache_dir as well

Reply:
hm/ do not know, I think `models_storage` represents exactly the purpose of the parameter whereas `cache_dir` is more ambiguous (like, what exactly will be cached)